http/http_response.hpp

```diff
@@ -153,7 +153,7 @@ struct Response
     {
         BMCWEB_LOG_DEBUG << this << " Clearing response containers";
         stringResponse.emplace(response_type{});
-        jsonValue.clear();
+        jsonValue = nullptr;
```

Comment:
jsonValue is an object not a pointer right? should't it be like jsonValue=nlohmann::json();

Reply:
I think both ways can be used - https://github.com/nlohmann/json/issues/2156
Here, we're trying to sync up with what upstream does - https://github.com/openbmc/bmcweb/blob/9970e93f794d0110de436828775e333e81330ad5/http/http_response.hpp#L194